implementations/rust/ockam/ockam_api/src/cli_state/repositories.rs

```diff
@@ -42,8 +42,11 @@ impl CliState {
         Arc::new(NodesSqlxDatabase::new(self.database()))
     }
 
-    pub(super) fn policies_repository(&self) -> Arc<dyn PoliciesRepository> {
-        Arc::new(PolicySqlxDatabase::new(self.database()))
+    pub(super) fn policies_repository(&self) -> Policies {
```

Comment:
Should be renamed to policies() probably

Reply:
done